clients/src/main/java/org/apache/kafka/clients/admin/DescribeTopicsOptions.java

```diff
@@ -47,8 +49,25 @@ public DescribeTopicsOptions includeAuthorizedOperations(boolean includeAuthoriz
         return this;
     }
 
+    public DescribeTopicsOptions useDescribeTopicsApi(boolean useDescribeTopicsApi) {
+        this.useDescribeTopicsApi = useDescribeTopicsApi;
+        return this;
+    }
```

Comment:
Can we add some comments here for a developer to know _why_ to use the topics API or not?

Reply:
Removed this option.